### ISSUE TITLE
NTR:  Beau's lines

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6978]
@pnrobinson - not sure if this an appropriate term? I was just reading an article about this:
https://www.washingtonpost.com/lifestyle/wellness/covid-nails-symptoms-beaus-lines/2021/05/11/c449243e-b1b4-11eb-9059-d8176b9e3798_story.html

**Preferred term label:**
Beau's lines

**Definition (free text, please give PubMed ID)**
Deep grooved lines that run from side to side on the fingernail or the toenail. They may look like indentations or ridges on the nail plate.

Source:
Freedberg, et al. (2003). Fitzpatrick's Dermatology in General Medicine. (6th ed.). McGraw-Hill. ISBN 0-07-138076-0. (from https://en.wikipedia.org/wiki/Beau%27s_lines)

**Parent term (use hpo.jax.org/app)**
Abnormality of the nail HP:0001597

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Comment by pnrobinson** [comment]
Thanks, this does seem like a good hpo term, see also https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7504889/

**Comment by nicolevasilevsky** [comment]
Great! I will add it now :)
